src/ui/ResultActions/ResultActionsMenu.ts

```diff
@@ -0,0 +1,126 @@
+import { Component } from '../Base/Component';
+import { ComponentOptions } from '../Base/ComponentOptions';
+import { IResultsComponentBindings } from '../Base/ResultsComponentBindings';
+import { $$ } from '../../utils/Dom';
+import { IQueryResult } from '../../rest/QueryResult';
+import { Assert } from '../../misc/Assert';
+import { Initialization } from '../Base/Initialization';
+import * as _ from 'underscore';
+import { exportGlobally } from '../../GlobalExports';
+import 'styling/_ResultActionsMenu';
+
+export interface IResultActionsMenuOptions {
+  openOnMouseOver?: boolean;
+}
+
+/**
+ * The ResultActionsMenu is a small floating slack-like result actions menu.
+ * It is designed to contain "buttons" that execute an action on a result when pressed.
+ * For example you can include the Quickview inside that menu or the AttachToCase.
+ *
+ * Example of how to use it in a template:
+ *
+ * ```html
+ * <script type="text/html" class="result-template" [...]
+ *   <div class="coveo-result-frame">
+ *     <div class="CoveoResultActionsMenu">
+ *       <div class="CoveoAttachToCase" data-display-text="false"></div>
```

Comment:
This is not a standard component ("AttachToCase"). We can leave it here, but maybe we can add explanation that this is a component available only in Salesforce or Dynamics ?

Reply:
J'ai reformulé la dernière ligne de la description pour: `For example you can include the Quickview inside that menu or result action components, like the AttachToCase, from the Salesforce or Dynamics integration.`. Pour le reste, je pelte ça dans la cours à @amoreauCoveo

Reply:
ça reste que avoir un component "AttachToCase" qui existe pas dans le default JSUI c'est pas trippant

you can remove this line and keep only the quickview